src/NextGen/DonationForm/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -167,12 +236,12 @@ protected function mapGenericBlockAttributesToNode(Node $node, BlockModel $block
                 $node->required($block->attributes['isRequired']);
             }
 
-            if($block->hasAttribute('displayInAdmin') && $block->attributes['displayInAdmin']) {
+            if ($block->hasAttribute('displayInAdmin') && $block->attributes['displayInAdmin']) {
                 $node->displayInAdmin = $block->attributes['displayInAdmin'];
             }
 
             /** TODO: ask kyle about $node->showInReceipt() */
```

Comment:
I had added a macro to implement `shouldDisplayInReceipt`.

```php
        Field::macro('shouldDisplayInReceipt', function() {
            return isset($this->displayInReceipt) && $this->displayInReceipt;
        });
```

However, it seems that the Fields API uses the `ShowInReceipt` trait, so we should consider refactoring `displayInReceipt` to `showInReceipt` so that we can use the `showInReceipt()` and `shouldShowInReceipt()` methods.

Reply:
yes would be a good one for cooldown